docs/docs/getting-started/05-examples/03-mock-hardware-example.md

```diff
@@ -0,0 +1,206 @@
+---
+sidebar_position: 3
+---
+
+# An application with hardware
+
+## Package requirements
+
+This example requires the Universal Robots collection which includes example URDF content.
+
+```toml title="aica-package.toml"
+#syntax=ghcr.io/aica-technology/app-builder:v1
+
+[packages]
+"@aica/base" = "v2.1.0"
+
+"@aica/collections/ur-collection" = "v3.0.4"
+```
+
+## URDF Hardware Manager
+
+After starting the application container, go to `localhost:8080/dev/hardware` to open the Hardware Manager page of
```

Comment:
Can we not access this through the start page?

Reply:
Just jumping in on this segment to ask, would it make sense that localhost references are actually links? For example, instead of:

`localhost:8080/dev/hardware`

you get:

[http://localhost:8080/dev/hardware](http://localhost:8080/dev/hardware)

Both are highlighted in the docs, but the latter seems slightly more functional. What do you think?

Reply:
![image](https://github.com/aica-technology/api/assets/71256590/cd0df64f-cd26-4275-94f7-e1da085050bd)

I'd rather refer to this

Reply:
I'll add both, thanks